src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/IWindow.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    /// <summary>
+    /// Interface for window
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public interface IWindow
+    {
+        /// <summary>
+        /// Gets window resource ID
+        /// </summary>
+        /// <returns></returns>
+        int GetResId();
```

Comment:
I recomment to use property
```suggestion
        int ResourceId {get;}
```

Reply:
Done